docs/data/sdks/marketing-analytics-browser/index.md

```diff
@@ -0,0 +1,93 @@
+
+amplitude.init("YOUR_API_KEY_HERE");
+</script>
+```
+
+## Usage
+
+### Initialize the SDK
+
+--8<-- "includes/sdk-httpv2-notice.md"
+
+--8<-- "includes/sdk-ts-browser/init.md"
+
+### Configuration
+
+--8<-- "docs/data/sdks/ts-browser/shared-configurations.md"
+|`trackingOptions`| Optional. `TrackingOptions` | [Learn more about tracking options](https://www.docs.developers.amplitude.com/data/sdks/marketing-analytics-browser/#optional-tracking/). |
+|`attribution.disabled`| Optional. `boolean` | Disable the attribution tracking, attribution is enabled by default |
+|`attribution.excludeReferrers`|  Optional. `string[]` | Exclude the attribution traccking for the provided referrers string |
+|`attribution.initialEmptyValue`| Optional. `string` | Reset the sessionId on a new campaign, Default value is `EMPTY` |
+|`attribution.resetSessionOnNewCampaign`| Optional. `boolean` | Reset the sessionId on a new campaign, will not create a new session for new campaign by default |
+|`pageViewTracking.trackOn`| Optional. `attribution` or `() => boolean` | `attribution` - Fire a page view event attribution information changes. `undefined` - Fire a page view event on page load or on history changes for single page application, default behavior. `() => boolean` - Fire a page view events based on a trackOn functions|
+|`attribution.pageViewTracking.trackHistoryChanges`  | Optional. `pathOnly` or `all` | Track the page view only on the path changes, track `all` url changes by default|
+
+--8<-- "includes/sdk-ts-browser/basic-ts-functions.md"
```

Comment:
When looking at the page, it kind feels long one and the Web Attribution part falls on the bottom, I wonder if we could leave the basic-ts-functions out and only the ones with marketing analytics, so we don't duplicate the docs. Any thoughts? cc @kevinpagtakhan @dnshi

Reply:
Discuss with @kevinpagtakhan about this. His point is customers will try to seek all the information on one page. But I do think market analytics stuff is not stand out. Maybe we can move to the first section?